src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java

```diff
@@ -837,7 +837,7 @@ public ByteArrayArray createComplexProperty() {
    * Defines MSExchangeCertificate Property **
    */
 
-  public static PropertyDefinition MSExchangeCertificate =
+  public static final PropertyDefinition MSExchangeCertificate =
```

Comment:
Completely agree! I just didn't want to touch a lot of sources. I can fix it in this PR or in separate PR. What do you prefer?